requirements-dev.txt

```diff
@@ -1,19 +1,19 @@
 -r requirements.txt
 
 # For tests
-git+https://github.com/alphagov/digitalmarketplace-test-utils.git@1.4.0#egg=digitalmarketplace-test-utils==1.4.0
-coverage==3.7.1
-flake8-per-file-ignores==0.4.0
+coverage==4.5.2
+coveralls==1.5.1
 flake8==3.5.0
-freezegun==0.3.8
-hypothesis==2.0.0
+flake8-per-file-ignores==0.6.0
+freezegun==0.3.11
+hypothesis==3.82.1
```

Comment:
As discussed offline - let's try bumping this back down to v3.44.21 which will hopefully avoid the `attrs` problem.